Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcCopyClient.cs

```diff
@@ -256,7 +263,7 @@ private async Task<(long Chunks, long Bytes)> StreamContentWithCompressionAsync(
             {
                 using (Stream decompressedStream = new GZipStream(grpcStream, CompressionMode.Decompress, true))
                 {
-                    await decompressedStream.CopyToAsync(targetStream, _bufferSize, ct).ConfigureAwait(false);

```

Comment:
Minor:  I'm not a fan of your crusade against ConfigureAwait(false). I find it very annoying that BCL choose the wrong default, but I also really don't look forward to putting back on a bunch of ConfigureAwait(false) when someone wants to re-use our code in an app with a UI.

Reply:
I've added an analyzer into ErrorProne.NET that is used in the solution right now. So if we'll change our mind, it will be one line change (for instance [here](http://index/repos/domino?query=%60donotuseconfigureawait&rightProject=BuildXL.Cache.ContentStore.Distributed&file=Properties%5CAssemblyInfo.cs&line=6)) and then we'll run a global fixer to use `ConfigureAwait(false)` everywhere in the entire project.

That's why I think consistency is actually important here: we can either use `ConfigureAwait(false)` everywhere or not use it at all. And this decision is controlled per project level.